test/index.js

```diff
@@ -0,0 +1,105 @@
+var chai = require('chai');
```

Comment:
in the coming days, lib API will change  (https://github.com/alt-j/react-server/issues/5), so you can cover only utils